CHANGELOG.md

```diff
@@ -3410,3 +3420,5 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 [v1.1.0]: https://github.com/nasa/cumulus/compare/v1.0.1...v1.1.0
 [v1.0.1]: https://github.com/nasa/cumulus/compare/v1.0.0...v1.0.1
 [v1.0.0]: https://github.com/nasa/cumulus/compare/pre-v1-release...v1.0.0
+
+[thin-egress-app]: <https://github.com/asfadmin/thin-egress-app> "Thin Egress App"
```

Comment:
Nit: 
Does this belong here?    The heading is weird already on the changeset section (should we have added a ## [v2.0.0] header when we went v2?), but this is a link to a ... not any more quasi-dependency.

Reply:
This is another way of doing links in Markdown that allows you to only specify the destination of a link once: https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
🤦  👍